### ISSUE TITLE
New Paper (Other): SARS and MERS: Recent insights into emerging coronaviruses

### ISSUE THREAD

**Issue by rando2** [greenelab/covid19-review#54]
# Paper Background
## Please paste a link to the paper or a citation here:

https://doi.org/f8v5cv

## What is the paper's [DOI](https://library.uic.edu/help/article/1966/what-is-a-doi-and-how-do-i-use-them-in-citations)?

DOI: 10.1038/nrmicro.2016.81

### Is this paper primarily relevant to Background or Pathogenesis? **(If it is primarily relevant to diagnostics or therapeutics, please go back and use the correct template)**

- [x] Background
- [x] Pathogenesis
- [ ] Other: 

## Please list some keywords (3-10) that help identify the relevance of this paper to COVID-19

* transmission
* pathogenesis
* SARS
* MERS
* coronavirus
* review

## Which areas of expertise are particularly relevant to the paper (put an x in the brackets [x])?

- [x] virology
- [ ] epidemiology
- [ ] biostatistics
- [x] immunology
- [ ] pharmacology

## Suggested Reviewers (feel free to suggest yourself)
@rmrussell (who initially added this citation to the text)

 
--------------------------------------------------------------------------------------------------------
# Questions for Reviewers to Address
Reviewers, please copy and paste the section below into a comment and fill it in.

## Suggested questions to answer about each paper:
- What did they analyze?
- What methods did they use?
- Does this paper study COVID-19, SARS-CoV-2, or a related disease and/or virus?
- What is the main finding (or a few main takeaways)?
- What does this paper tell us about the background and/or diagnostics/therapeutics for COVID-19 / SARS-CoV-2?
- Do you have any concerns about methodology or the interpretation of these results beyond this analysis?

**Comment by aadattoli** [comment]
Hi, I am for sure interested, just wanted to know whether there is a template that I can modify/add sections and so on...sorry I am new to this system!ehehe

**Comment by aadattoli** [comment]
Dear All,

I edited the section about treatments and I added the tolicizumab drug analysis/description with relative literature. I am now working on the antiviral section. Can you guys see my changes? Please let me know your feedback.

**Comment by agitter** [comment]
@aadattoli yes we see the changes in #75.  I left a couple initial formatting suggestions and then someone else can help review the content.

**Comment by eberwine** [comment]
Hi Ada,  I wonder if there is any literature on combinatorial use of the drugs you highlight in the review for other diseases or indications.  I will see what I can find.

**Comment by aadattoli** [comment]
Dear Jim, yes you are right, there are evidence of IFN-beta being more effective in combination with lopinavir, ritonavir and ribravin. I will add it.
